db/migrate/20151101224210_generate_token_for_existing_users.rb

```diff
@@ -0,0 +1,7 @@
+class GenerateTokenForExistingUsers < ActiveRecord::Migration
+  def change
+    User.all.each do |user|
```

Comment:
You dont have to revert this logic once you have run it now. Since its 1 time action.

To run your new custom rake task just do `heroku run rake namespace:task_name` to migrate data on heroku.